pkg/collector/worker/worker.go

```diff
@@ -29,6 +30,17 @@ const (
 	pollingInterval = 15 * time.Second
 )
 
+// The worker utilization is also reported via expvars, but it emits one metric
+// for each worker, which is a bit inconvenient to use because the number of
+// workers might be different on every Agent. With telemetry, we can use a
+// single metric and put the worker name in a tag.
+var workerUtilization = telemetry.NewGauge(
+	"workers",
+	"utilization",
```

Comment:
The final metric name would be `datadog.agent.workers.utilizaiton`, but it may not be specific enough, as there are workers in several different agent components.

```suggestion
	"collector",
	"worker_utilization",
```